### PR TITLE
Covid pages QA

### LINKED ISSUE #5533: Covid pages QA
Changes to the hard-coded Covid pages:

- [x] https://wellcomecollection.org/covid-welcome-back#book-your-ticket
Use different button style. The primary button should only be used when we're triggering a primary action (e.g. if this button linked directly to the ticket booking service). Should we try this instead? 
Text should read `Book your ticket`

Screenshot 2020-09-18 at 17 07 08

### PR DESCRIPTION
**Pull request by davidpmccormick**
Closes #5533

Copy/design amends for new covid pages.

### CODE REVIEW

**Comment by kenoir** [comment]
@davidpmccormick you'll need to merge master to get Buildkite to pass here.

File: content/webapp/pages/covid-book-your-ticket.tsx [hunk 1]
```diff
@@ -200,21 +194,27 @@ const CovidWeAreOpenPage = () => {
...
-                        book just one RMR slot each week.
+                        Materials Room.{' '}
+                        <strong>
+                          To start with, we’ll ask visitors to book just one RMR
+                          slot each week
+                        </strong>
+                        .
                       </li>
                       <li>
                         After you’ve booked your ticket, request up to 10 items
```
Comment (jamesgorrie) [inline content/webapp/pages/covid-book-your-ticket.tsx:209]: What's the  rule on `.` on list items? Seems to be applied differently in different places.

File: content/webapp/pages/covid-welcome-back.tsx [hunk 2]
```diff
@@ -275,29 +284,39 @@ const CovidWelcomeBackPage = () => {
...
+                    </li>
+                    <li>
+                      Providing optional mask-exempt badges from{' '}
+                      <a href="https://www.euansguide.com/news/face-mask-exempt-badges/">
+                        Euan’s Guide
+                      </a>
+                      .
+                    </li>
+                    <li>
+                      Participating in NHS Track and Trace through our online
```
Comment (jamesgorrie) [inline content/webapp/pages/covid-welcome-back.tsx:317]: Should this link to a form?

File: content/webapp/pages/covid-book-your-ticket.tsx (hunk 1 above)
Comment (davidpmccormick) [reply content/webapp/pages/covid-book-your-ticket.tsx:209]: everywhere where lists are sentences we add stops (there's just one list where this isn't the case, I believe)

File: content/webapp/pages/covid-welcome-back.tsx (hunk 2 above)
Comment (davidpmccormick) [reply content/webapp/pages/covid-welcome-back.tsx:317]: will check

**Review by jamesgorrie — APPROVED** [review]

File: content/webapp/pages/covid-book-your-ticket.tsx (hunk 1 above)
Comment (jennpb) [reply content/webapp/pages/covid-book-your-ticket.tsx:209]: Full sentences in bulleted lists get a `.`
Phrases in bulleted lists only get a `.` on the final bulleted phrase.

Comment (jamesgorrie) [reply content/webapp/pages/covid-book-your-ticket.tsx:209]: Great! I've always wanted a rule for this in docs.

Comment (davidpmccormick) [reply content/webapp/pages/covid-book-your-ticket.tsx:209]: image

…no final `.` after the final bulleted phrase in the doc from Kate

Comment (jennpb) [reply content/webapp/pages/covid-book-your-ticket.tsx:209]: Yes, I would.

**Pull request merged**